### ISSUE TITLE
Unable to use wp cli media regenerate in stateless mode

### ISSUE THREAD

**Issue by Nks** [udx/wp-stateless#488]
If we will try to run command with [wp cli](https://wp-cli.org/) `wp media regenerate --skip-delete --yes` to sync missing media to the server (instead of using sync option provided by plugin) it will cause the fatal error.

```PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function wpCloud\StatelessMedia\Utility::store_can_delete_attachment(), 2 passed and exactly 3 expected in /mnt/disks/main_disk/domains/.................../content/web/app/plugins/wp-stateless/lib/classes/class-utility.php:572
Stack trace:
#0 [internal function]: wpCloud\StatelessMedia\Utility::store_can_delete_attachment(Array, Array)
#1 /mnt/disks/main_disk/domains/.................../content/web/app/plugins/wp-stateless/lib/classes/class-bootstrap.php(1524): call_user_func_array(Array, Array)
#2 /mnt/disks/main_disk/domains/.................../content/web/wp/wp-includes/class-wp-hook.php(287): wpCloud\StatelessMedia\Bootstrap->__call('store_can_delet...', Array)
#3 /mnt/disks/main_disk/domains/.................../content/web/wp/wp-includes/plugin.php(206): WP_Hook->apply_filters(Array, Array)
#4 phar:///usr/bin/wp/vendor/wp-cli/media-command/src/Media_Command.php(828): apply_filters('intermediate_im...', Array, Array)
#5 phar:///usr/bin/wp/vendo in /mnt/disks/main_disk/domains/.................../content/web/app/plugins/wp-stateless/lib/classes/class-utility.php on line 572
Fatal error: Uncaught ArgumentCountError: Too few arguments to function wpCloud\StatelessMedia\Utility::store_can_delete_attachment(), 2 passed and exactly 3 expected in /mnt/disks/main_disk/domains/.................../content/web/app/plugins/wp-stateless/lib/classes/class-utility.php:572
Stack trace:
#0 [internal function]: wpCloud\StatelessMedia\Utility::store_can_delete_attachment(Array, Array)
#1 /mnt/disks/main_disk/domains/.................../content/web/app/plugins/wp-stateless/lib/classes/class-bootstrap.php(1524): call_user_func_array(Array, Array)
#2 /mnt/disks/main_disk/domains/.................../content/web/wp/wp-includes/class-wp-hook.php(287): wpCloud\StatelessMedia\Bootstrap->__call('store_can_delet...', Array)
#3 /mnt/disks/main_disk/domains/.................../content/web/wp/wp-includes/plugin.php(206): WP_Hook->apply_filters(Array, Array)
#4 phar:///usr/bin/wp/vendor/wp-cli/media-command/src/Media_Command.php(828): apply_filters('intermediate_im...', Array, Array)
#5 phar:///usr/bin/wp/vendo in /mnt/disks/main_disk/domains/.................../content/web/app/plugins/wp-stateless/lib/classes/class-utility.php on line 572
```

As wp-cli is a part of the wordpress I think it should works without fatal error. 

Here the part from Media_Command.php (row 828):

```
if ( ! $is_pdf && is_array( $metadata ) ) {
			// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound -- Calling native WordPress hook.
			$sizes = apply_filters( 'intermediate_image_sizes_advanced', $sizes, $metadata );
		}
```

**Comment by planv** [comment]
@Nks New functionality for sync media via CLI will be included in the next release.

**Comment by planv** [comment]
This has been fixed in 3.0

List of available commands - https://github.com/udx/wp-stateless/wiki/WP-CLI

Closing the ticket.
